app-editors/bvi/bvi-1.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="display-oriented editor for binary files, based on the vi texteditor"
+HOMEPAGE="http://bvi.sourceforge.net/"
+SRC_URI="mirror://sourceforge/bvi/${P}.src.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~sparc64-solaris ~x86-solaris"
```

Comment:
Please keep `KEYWORDS` sorted, no need to do it manually, `ekeyword` will do it for you.

Reply:
I ran `ekeyword all bvi-1.4.0.ebuild && ekeyword ~all bvi-1.4.0.ebuild` and it put the keywords in the same order they already are (just running it with `~all` didn’t do anything because they were already all `~`). What order do you want? Strict lexicographical? Non-prefix before prefix?

Reply:
I think the current repoman version complains about unsorted keywords, so if it doesn't complain, it's fine.

Reply:
Repoman 2.3.11 has no complaints about the new ebuild (it has some complaints about the older ones, but not 1.4.0).